STL_Extension/include/CGAL/STL_Extension/internal/parameters_interface.h

```diff
@@ -339,6 +339,8 @@ CGAL_add_named_parameter_with_compatibility(input_features_param_t, input_featur
 CGAL_add_named_parameter_with_compatibility(edge_size_param_t, edge_size_param, edge_size)
 CGAL_add_named_parameter_with_compatibility_cref_only(edge_sizing_field_param_t, edge_sizing_field_param, edge_sizing_field)
 CGAL_add_named_parameter_with_compatibility(edge_min_size_param_t, edge_min_size_param, edge_min_size)
+CGAL_add_named_parameter_with_compatibility_cref_only(edge_distance_sizing_field_param_t, edge_distance_sizing_field_param, edge_distance_sizing_field)
+CGAL_add_named_parameter_with_compatibility_cref_only(edge_distance_param_t, edge_distance_param, edge_distance)
```

Comment:
I do not understand why there is `edge_distance_sizing_field`. That parameter is not documented, and seems equivalent to `edge_distance`.

@janetournois Do you know why we have all those parameters named `*_sizing_field`?

Reply:
Conclusion of a live discussion with @lrineau and @ange-clement :
* there is no need for the `edge_distance_field` parameter, `edge_distance` is enough to deal with constant and functional versions of the parameter
* the other `*_sizing_field` parameters have been there since CGAL-3.8 but never documented and probably never used. We will submit another PR to remove them, to make the code shorter/simpler and reduce the number of parameters